modules/support-insecure-tls-connections.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="support-insecure-tls-connections_{context}"]
+= Using must-gather with insecure TLS connections
+
+If a custom CA certificate is used, the `must-gather` pod fails to grab the output for `velero logs/describe`. To use `must-gather` with  insecure TLS connections, you can pass the `gather_without_tls` flag to the `must-gather` command.
```

Comment:
Is the second usage of `must-gather` referring to the tool or the command? It could be helpful to pair the literal with a noun.

Reply:
Good point. I suggest using "tool" as we do elsewhere in the documentation.